### DISCUSSION TITLE
Add Guatemala to list of countries please

### DISCUSSION THREAD

**Discussion by MoveLNsats** [sovenor/bitcoin-rocks#52] [Ideas]
When people scan your QR code it would be nice if they could see Guatemala on the list.

I post lots of stickers in Guatemala in 4 big cities and 'bitcoin lake Atitlan"

Thank you
Frigfren

**Comment by ObjectifMoon21** [comment]
Hi @MoveLNsats !

Now [bitcoin rocks](http://bitcoin.rocks) is available in Spanish, you can select the language in the orange banner on the top of the page.

but maybe you're talking about adding Guatemela currency to the inflation page. Good idea ! Let's see if @sovenor can do it?!

@MoveLNsats I see the official currency is the Quetzal (GTQ), by any chance, do you have sources on inflation in Guatemala over the last 5 years and the amount of money supply over the last few years?

**Comment by MoveLNsats** [comment]
adding Guatemela currency to the inflation page. Good idea ! Let's see ***@***.***(https://github.com/sovenor)

Yes please

Phil Deck
You can buy Bitcoin here
https://river.com/signup?r=G4DDJHKV

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
